src/base/ElementInterface.php

```diff
@@ -881,6 +881,15 @@ public function canDelete(User $user): bool;
      */
     public function canDeleteForSite(User $user): bool;
 
+    /**
+     * Returns whether the given user is authorized to move this element.
+     *
+     * @param User $user
+     * @return bool
+     * @since 5.0.0
+     */
+    public function canMove(User $user): bool;
```

Comment:
It seems a little weird to introduce the concept of “moving” to all element types, when entries are the only one that would actually know what to do with it.

Reply:
As per our chat, this might end up being a way to go.
But, for now, I have reverted those changes and used a different approach where I added the ability to further customise element chip & card attributes, and I’m adding `data-movable` via those.